apidoc/Titanium/UI/ListItem.yml

```diff
@@ -708,3 +708,12 @@ properties:
         By default the background color of the row action is defined by the style applied. Use this property to 
         override the default background color of the row action.
     optional: true
+
+  - name: image
+    type: String
+    summary: The image/icon of the row action.
+    description: |
+        You can combine this property with the `color` property to apply an icon + background color in combination.
+        Note: When using this property, the `title` property is ignored.
+        Available since Titanium SDK 12.1.0
```

Comment:
since it is a new property you can use
```
    since: 12.1.0
    platforms: [iphone, ipad, macos]
```

instead of writing it in the description.

Reply:
at least in a quick build it showed up in the generated HTML docs:
![Screenshot_20230115_183141](https://user-images.githubusercontent.com/4334997/212557037-4b1bbee5-b98d-4b6c-a8ef-1b0a007d8e8a.png)
should show up correctly in https://titaniumsdk.com/api/structs/rowactiontype.html too I hope :smile:

Reply:
Okay, very interesting! I'll update it and we're good to go